File: src/java/src/quickfix/fix40/Heartbeat.java
  package quickfix.fix40;
  import quickfix.FieldNotFound;
  import quickfix.Group;
  import quickfix.field.*;

  public class Heartbeat extends Message
  {
    public Heartbeat()
    {
      getHeader().setField(new MsgType("0"));
    }

    public void set(quickfix.field.TestReqID value)
    { setField(value); }
    public quickfix.field.TestReqID get(quickfix.field.TestReqID  value)
      throws FieldNotFound
    { getField(value); return value; }
    public quickfix.field.TestReqID getTestReqID() throws FieldNotFound
    { quickfix.field.TestReqID value = new quickfix.field.TestReqID();
      getField(value); return value; }
    public boolean isSet(quickfix.field.TestReqID field)
    { return isSetField(field); }
    public boolean isSetTestReqID()
    { return isSetField(112); }
}

File: src/.NET/fix41/Heartbeat.cs
namespace QuickFix41
{

  public class Heartbeat : Message
  {
    public Heartbeat() : base(MsgType()) {}
    static QuickFix.MsgType MsgType() { return new QuickFix.MsgType("0"); }

    public void set(QuickFix.TestReqID value)
    { setField(value); }
    public QuickFix.TestReqID get(QuickFix.TestReqID  value)
    { getField(value); return value; }
    public QuickFix.TestReqID getTestReqID()
    { QuickFix.TestReqID value = new QuickFix.TestReqID();
      getField(value); return value; }
    public bool isSet(QuickFix.TestReqID field)
    { return isSetField(field); }
    public bool isSetTestReqID()
    { return isSetField(112); }

  };

}

